build/Helix/RunTestsInHelix.proj

```diff
@@ -13,268 +13,268 @@
     
     <HelixWorkItem Include="InteractionTests.ColorPickerTests">
       <Timeout>00:20:00</Timeout>
-      <Command>call %HELIX_CORRELATION_PAYLOAD%\runtests.cmd /name:Windows.UI.Xaml.Tests.MUXControls.InteractionTests.ColorPickerTests.*</Command>
+      <Command>call %HELIX_CORRELATION_PAYLOAD%\runtests.cmd .\InstallTestAppDependencies.ps1 "MUXControls.Test.dll MUXControlsTestApp.appx" /name:Windows.UI.Xaml.Tests.MUXControls.InteractionTests.ColorPickerTests.*</Command>
```

Comment:
I would rather keep the HelixWorkItem definitions here "dumb" and make runtests.cmd smarter.

I already added a comment about making InstallTestAppDependencies.ps1 more general purpose, so with that change, it should be possible to remove that from the Command.

runtests.cmd can probably also be updated so that it doesn't need to specify the test dlls/appxs. I think if you just list all the dlls/appx all the time everything should work. We don't have duplicated test names across these binaries, so the queries should work as expected. As far as I remember, te.exe does not fail if you pass it names files that don't exist - though it does write a warning/error to the log which might be confusing. So an improvement there is to have the script construct the set of test binaries by checking for their existence first.